templates/admin/group/group_property_editor.html.ep

```diff
@@ -187,6 +187,65 @@
                     <p class="properties-status"></p>
                 </div>
             </div>
+            <div class="form-group row">
```

Comment:
Of course the GUI is still under development anyways, but I'd already say this is the wrong place. I would put this outside the properties box where the current table is. This obviously affects the code of the JavaScript function `toggleEdit()` as well.

---

Another problem is that this change would be immediately visible to all admins when we merge and deploy the PR. I'd say in this state it isn't ready.

Reply:
I was under the impression it should be visible to everyone so people can play with it and see how it works before actually making changes with it.

Hmmm maybe a sort of "Switch to YAML editor" that swaps the table with the editor? The switch would go away in the future when the editor becomes the primary UI.

Reply:
> Hmmm maybe a sort of "Switch to YAML editor" that swaps the table with the editor? The switch would go away in the future when the editor becomes the primary UI.

That would make sense. It is just that in the current form it is a little bit confusing that the new UI is mixed within the old UI without further explanation.

By the way, I'm also against letting people play with it *in production* unless the preview feature is implemented.

Reply:
Unless? The editor only uses the preview mode. You can't make any changes there.

Reply:
Good. Then it is not a big deal actually.

Reply:
I played around with it a little bit more and noticed that the new UI is also shown on parent group pages leaving those unusable. This change should *not* affect parent groups at all.

Reply:
As discussed, I moved the new editor out of the properties and it's actually swapped in via a toggle button.